shared/scss/views/_tracker-networks.scss

```diff
@@ -1,6 +1,12 @@
-.tracker-networks .hero__icon {
-    width: 58px;
-    height: 65px;
+.tracker-networks {
+    .hero__icon {
+        width: 58px;
+        height: 65px;
+    }
+    .hero__subtitle {
+        height: 40px;
```

Comment:
What do you think about reducing this to 32px? It'll help keep the margins a bit more consistent, and leave less empty space when there's only one line of text.

![screenshot from 2018-02-02 12-27-34](https://user-images.githubusercontent.com/1431577/35733417-3b8fc392-0815-11e8-9d37-c31e262c09bd.png)